ui-ngx/src/app/shared/models/dashboard.models.ts

```diff
@@ -126,6 +126,7 @@ export interface DashboardConfiguration {
 export interface Dashboard extends DashboardInfo {
   configuration?: DashboardConfiguration;
   dialogRef?: MatDialogRef<any>;
+  assignedCustomerIds?: Array<string>;
```

Comment:
It might be better to create a custom data type in the file dashboards-table-config.resolver.ts